packages/config/templates/package.template.json

```diff
@@ -3,7 +3,7 @@
         "commit:lint": "commitlint --edit $1",
         "lint": "eslint $(git diff-index --name-only --diff-filter=d HEAD | grep \"[\\.js|\\.jsx|\\.tsx|\\.ts]$\" | xargs)",
         "lint:fix": "npm run lint -- --fix",
-        "prepare": "husky install",
+        "prepare": "npm run theme; husky install",
         "update:config": "npx @ferbab/config"
     },
     "lint-staged": {
```

Comment:
"npm run lint&fix" not working

"npm run lint:fix" instead ?

Reply:
fix